.github/CODEOWNERS

```diff
@@ -120,7 +120,7 @@
 /types/amplify/                                         @joeriks
 /types/amplify-deferred/                                @joeriks @laurentiustamate94
 /types/amplitude-js/                                    @Asido @danmana @HintikkaKimmo
-/types/amplitude-js/v4/                                 @Asido @danmana @HintikkaKimmo
+/types/amplitude-js/vv4/                                @Asido @danmana @HintikkaKimmo
```

Comment:
This looks like it could be caused by my changes in https://github.com/microsoft/types-publisher/pull/723